api/controllers/job.py

```diff
@@ -33,30 +34,33 @@ def get_jobs_on_home_page(query: s.JobHomePage, current_user: m.User, db: Sessio
         sa.select(m.Job).join(m.SavedJob).where(m.SavedJob.user_id == current_user.id)
     ).all()
 
+    recommended_jobs_out = [
+        s.JobCard(
+            location=s.LocationStrings(
+                uuid=job.location.uuid,
+                name=job.location.region[0].name_ua if query.lang == CFG.UA else job.location.region[0].name_en,
+            ),
+            is_saved=job in user_saved_jobs,
+            **pop_keys(job.__dict__, ["location"]),
+        )
+        for job in recommended_jobs
+    ]
+    jobs_near_you_out = [
+        s.JobCard(
+            location=s.LocationStrings(
+                uuid=job.location.uuid,
+                name=job.location.region[0].name_ua if query.lang == CFG.UA else job.location.region[0].name_en,
+            ),
+            is_saved=job in user_saved_jobs,
+            **pop_keys(job.__dict__, ["location"]),
+        )
+        for job in jobs_near_you
+    ]
+
     return s.JobsCardList(
         lang=query.lang,
-        recommended_jobs=[
-            s.JobCard(
-                location=s.LocationStrings(
-                    uuid=job.location.uuid,
-                    name=job.location.region[0].name_ua if query.lang == CFG.UA else job.location.region[0].name_en,
-                ),
-                is_saved=job in user_saved_jobs,
-                **pop_keys(job.__dict__, ["location"]),
-            )
-            for job in recommended_jobs
-        ],
-        jobs_near_you=[
-            s.JobCard(
-                location=s.LocationStrings(
-                    uuid=job.location.uuid,
-                    name=job.location.region[0].name_ua if query.lang == CFG.UA else job.location.region[0].name_en,
-                ),
-                is_saved=job in user_saved_jobs,
-                **pop_keys(job.__dict__, ["location"]),
-            )
-            for job in jobs_near_you
-        ],
+        recommended_jobs=recommended_jobs_out,
+        jobs_near_you=jobs_near_you_out,
```

Comment:
Same problem as with user list. This endpoint should not return language, and to lists. it should return one list on jobs, depending on request query params